Assets/Scripts/FpsMovement.cs

```diff
@@ -23,12 +23,11 @@ public class FpsMovement : MonoBehaviour
     [SerializeField]
     private bool _grounded = false;
 
-    [SerializeField]
-    private Rigidbody _rigidbody;
+    public Rigidbody rigidbody;
```

Comment:
nit: Suggest naming field `body` to not have warning about overriding `rigidbody` field from MonoBehaviour